README.md

```
@@ -2,7 +2,7 @@
 
 - [monkas](#monkas)
 - [Dependencies](#dependencies)
-  - [Quick install for debian/ubuntu based distors](#quick-install-for-debianubuntu-based-distors)
+  - [Quick installation for debian/ubuntu based distros](#quick-install-for-debianubuntu-based-distors)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)